tool/stats_lib.dart

```diff
@@ -54,19 +61,16 @@ class Config {
   static final Config commonMarkConfig = Config._(
     'common_mark',
     'http://spec.commonmark.org/0.28/',
-    null,
   );
   static final Config gfmConfig = Config._(
     'gfm',
     'https://github.github.com/gfm/',
-    ExtensionSet.gitHubFlavored,
```

Comment:
The `ExtensionSet.gitHubFlavored` does not work out here. Because `AutolinkExtensionSyntax` and `AutolinkSyntax` conflict to each other.

For example: 
https://github.github.com/gfm/#example-620
```markdown
http://example.com
```

Without `AutolinkExtensionSyntax`, the result is:
```html
<p>http://example.com</p>
```
With `AutolinkExtensionSyntax`, the result is:
```html
<p><a href="http://example.com">http://example.com</a></p>
```